mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
```

Comment:
The function `exports.purgeOrder` is defined twice in the codebase, which can lead to conflicts and unexpected behavior. This duplication should be removed, and each function should be uniquely named to avoid such issues.